### ISSUE TITLE
Database management system, plus features

### ISSUE THREAD

**Issue by saulgreenberg** [saulgreenberg/Timelapse#67]
Timelapse currently handles a single database - matching an image set - at a time.

An enhancement would be to add a dbms that handles all the databases created, e.g., a meta-timelapse. 

Possible features:
- if two or more analysts classify an image set, the dbms will compare the two resulting data sets for differences, and highlight those entries that differ for resolution.

**Comment by saulgreenberg** [comment]
Another possibility is to add a new program that will merge timelapse databases into a single one.

**Comment by saulgreenberg** [comment]
A user wrote:
I looked at a number of programs for working with camera trap data. Timelapse and Camelot were 2 that I found most useful and user friendly.. but they really do represent 2 very different roles within the spectrum. Timelapse is super easy to get going with quickly and has the most useful tools for working through photos. Camelot represented a lot more work on the front end, to organize a camera trap inventory and deployments/visits etc, but then after that effort, was able to do more of the add-on stuff (output reports or pre-formatted data for common analysis). It is more geared toward people concurrently managing numerous projects. I honestly never even got far enough in it to test the photo-parsing tools. But I really liked the concepts and the interface. Dan mentioned another program that was really GIS heavy that made maps, but I don’t know if I ever got that one to work on a test drive… I’ll try to remember the name & get back to you.
